hathor/indexes/manager.py

```diff
@@ -99,6 +101,8 @@ def _iter_all_indexes_with_filter(self) -> Iterator[Tuple[_IndexFilter, BaseInde
             yield _IndexFilter.ALL, self.addresses
         if self.tokens is not None:
             yield _IndexFilter.ALL, self.tokens
+        if self.utxo is not None:
+            yield _IndexFilter.ALL, self.utxo
```

Comment:
I avoided creating a new filter for that (because it would have to include non-voided blocks AND transactions). The index initialization function already filters-out voided tx/blocks so it works correctly when using `ALL`.